packages/vector_graphics_compiler/test/tessellator_test.dart

```diff
@@ -0,0 +1,345 @@
+import 'package:test/test.dart';
+import 'package:vector_graphics_compiler/src/svg/node.dart';
+import 'package:vector_graphics_compiler/src/svg/parser.dart';
+import 'package:vector_graphics_compiler/src/svg/resolver.dart';
+import 'package:vector_graphics_compiler/src/svg/tesselator.dart';
+import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
+
+import 'helpers.dart';
+
+void main() {
+  setUpAll(() {
+    if (!initializeTessellatorFromFlutterCache()) {
+      print('error in setup');
```

Comment:
Consider printing instructions about how to resolve this situation - unless they're already printed out, in which case just `fail()` here?